src/input/dependency.js

```diff
@@ -35,18 +35,33 @@ function dependencyStream(
       .concat(['.mjs', '.js', '.json', '.es6', '.jsx']),
     transform: [
       babelify.configure({
-        sourceMap: false,
+        sourceMaps: false,
         compact: false,
-        presets: [
-          require('babel-preset-env'),
-          require('babel-preset-stage-0'),
-          require('babel-preset-react')
-        ],
+        presets: ['@babel/preset-react', '@babel/preset-env'],
         plugins: [
-          require('babel-plugin-transform-decorators-legacy').default,
-          // Required to support webpack's System.import
-          // https://github.com/documentationjs/documentation/issues/578
```

Comment:
worth pointing out I couldn't find a Babel 7 version of the system-import-transformer plugin so I removed it.